packages/editor/src/components/visual-editor/index.js

```diff
@@ -389,7 +389,8 @@ function VisualEditor( {
 				{
 					'has-padding': isFocusedEntity || enableResizing,
 					'is-resizable': enableResizing,
-					'is-iframed': shouldIframe,
+					'is-iframed': ! disableIframe,
```

Comment:
I believe this should be unchanged:
```suggestion
					'is-iframed': shouldIframe,
```

Otherwise, this class will be absent for device previews which are indeed iframed.

For reference in trunk it was changed in #65887 and that was not backported for 6.7.

Reply:
Should we backport #65887 first? The problem #65887 is trying to solve is not new to WP 6.7, but to make it easier to backport this PR, it may be better to backport #65887 first.

Reply:
I don’t see that this depends on #65887. My previous comment was meant only to make it clear where the change came from. It was not part of #66706 and it looks like the change on the adjacent line probably made this a conflict against the `wp/6.7` branch.